### ISSUE TITLE
Only uploading final workflow outputs to the project

### ISSUE THREAD

**Issue by dolevrahat** [dnanexus/dxWDL#176]
Hello

I am running a multi-task workflow (specifically the[ GATK4 data processing workflow)](https://github.com/gatk-workflows/gatk4-data-processing).
Is there a possibility that only the final workflow outputs and not the task outputs will be uploaded to the project? I am using the --rerog flag and yet I see that many task outputs are still uploaded, leading to a considerable storage overhead.
Is there a way to only save the final workflow outputs?
Alternativily, is there some way you can recommend for automating deletion of intermediate output as part of/upon completion of the workflow?

Thanks in advacnce!

**Comment by orodeh** [comment]
Right. The `--reorg` flag puts all the intermediate outputs in a subfolder. You can delete this subfolder, avoiding the long term storage costs.
